modules/ROOT/pages/create-api-specification-studio.adoc

```diff
@@ -0,0 +1,41 @@
+= Create an API Project in Studio
+
+Anypoint Studio 7.7 and later enables you to create a RAML or OAS 2.0 and 3.0 API specification, or an API fragment in Studio and automatically synchronize it with Design Center.
```

Comment:
I was initially confused about how many things were being created and where, but I think this might clarify that a little:
```suggestion
Anypoint Studio 7.7 and later enables you to create a RAML API specification, an OAS 2.0 or 3.0 API specification, or an API fragment in Studio, and then to automatically synchronize it with Design Center.
```
Are these things synchronized TO WORK WITH Design Center or synchronized BY USING Design Center (or is this clear to your intended audience)?